lib/praxis/restful_doc_generator.rb

```diff
@@ -233,7 +238,7 @@ def write_types( versioned_types )
           # ...but not for links or link classes (there's no object container context if done alone!!)
           unless( type < Praxis::Links ) #TODO: again, why is this special?
             type_output[:example] = if example_data.respond_to? :render
-              example_data.render(view: :master)
+              nil #example_data.render(view: :master)
```

Comment:
need to put that back I guess...what is that we'd be rendering in terms of examples otherwise?...